core/ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/DevicePreviews.kt

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
Lint rule enforced me to make the file name as DevicePreviews, but it may be more clearer to make it file name something like MultipreviewAnnotations.kt and disable the lint warning by adding
// ktlint-disable filename